adv-java/src/main/java/dataModel/Login.java

```diff
@@ -0,0 +1,28 @@
+package dataModel;
+import sqlDao.LoginServlet; 
+import java.io.IOException;
+import javax.servlet.http.*;
+
+public class Login extends HttpServlet {
+	
+	String UserId,Password;
+
+public void doPost(HttpServletRequest req,HttpServletResponse res) throws IOException {
+		
+		UserId = req.getParameter("Luname");
+		Password = req.getParameter("Lupass");
+		
+		Userdata ud=new Userdata();
+		ud.setUserid(UserId);
+		ud.setPassword(Password);
+				
+		LoginServlet ls=new LoginServlet();
```

Comment:
This is not the correct use of servlet. You are not supposed to instantiate or make an object of the class that extends Servlet.